_alerts/2017-07-12-disruptive-forks.md

```diff
@@ -0,0 +1,86 @@
+---
+## This file is licensed under the MIT License (MIT) available on
+## http://opensource.org/licenses/MIT.
+
+title: "Potential disruptive chain forks"
+shorturl: "disruptive-forks"
+active: true
+banner: "Warning: Bitcoin may be unsafe to use on August 1st (click here for details)"
```

Comment:
Let's also change the title from:
`2017-07-12-disruptive-forks.md`

To:
`2017-07-12-chain-split.md`